sync-core/src/test/java/com/cloudant/sync/replication/BasicPushStrategyTest.java

```diff
@@ -185,9 +180,25 @@ private void oneDocCreatedThenUpdatedThenDeletedAndThenPush() throws Exception {
     }
 
     private void push(int expectedDocs) throws Exception {
+        this.push(expectedDocs,
+                BasicPushStrategy.DEFAULT_CHANGES_LIMIT_PER_BATCH,
+                BasicPushStrategy.DEFAULT_MAX_BATCH_COUNTER_PER_RUN,
+                BasicPushStrategy.DEFAULT_BULK_INSERT_SIZE,
+                BasicPushStrategy.DEFAULT_PUSH_ATTACHMENTS_INLINE);
+    }
+
+    private void push( int expectedDocs,
```

Comment:
Well there are different methods which override different things for example [here](https://github.com/cloudant/sync-android/blob/49609-replication-config-clean-up/sync-core/src/test/java/com/cloudant/sync/replication/BasicPushStrategyTest.java#L285) and [here](https://github.com/cloudant/sync-android/blob/49609-replication-config-clean-up/sync-core/src/test/java/com/cloudant/sync/replication/BasicPushStrategyTest.java#L212)